test/event_tracing.cpp

```diff
@@ -0,0 +1,65 @@
+//*****************************************************************************
+// Copyright 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <fstream>
+#include <iostream>
+#include <mutex>
+#include <stdlib.h>
+#include <vector>
+#include "nlohmann/json.hpp"
+
+#include "gtest/gtest.h"
+#include "ngraph/event_tracing.hpp"
+#include "ngraph/file_util.hpp"
+
+using namespace std;
+
+TEST(event_tracing, event_file)
+{
+    // Set the environment variable to ensure logging
+    ngraph::Event::enable_event_tracing();
```

Comment:
I think we should disable at the end of the test (or, even better, restore to the original state).

Reply:
Done